File: php-api-demo/ULB/create_ulb.php
<?php
require_once dirname(__FILE__) . '/../utils.php';
require_once dirname(__FILE__) . '/../config.php';

$ApiClient = new ApiClient();
$ApiClient->init(BASE_URL, PUBLIC_KEY, PRIVATE_KEY);

$Parameters = array('Action' => 'CreateULB',
    'Region' => 'cn-north-01',
    'ULBName' => 'test-ulb-api'
);
$response = $ApiClient->get($Parameters);
echo $response . "\n";

?>

File: api/python2.x-api-demo/ULB/create_ulb.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import sys
sys.path.append("..")
import json
from utils import ApiClient
from config import * 

if __name__=='__main__':
    arg_length = len(sys.argv)
    ApiClient = ApiClient(base_url, public_key, private_key)
    Parameters = {
            "Action":"CreateULB",
            "Region":"cn-north-01",
            "ULBName":"test-ulb-api"
            }
    response = ApiClient.get("/", Parameters)
    print json.dumps(response, sort_keys=True, indent=4, separators=(',', ': '))

